src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -363,4 +363,10 @@
   <data name="NamedTypeFullNameNotNullSuppressorJustification" xml:space="preserve">
     <value>The 'Type.FullName' property is never null when using the syntax 'typeof(T).FullName'</value>
   </data>
+  <data name="BlankLinesDescription" xml:space="preserve">
+    <value>Avoid multiple blank lines.</value>
```

Comment:
Either delete this resource + change the descriptor to pass null description OR make this string more informative over the message.